test/unit/codegen/codegen_coreneuron_cpp_visitor.cpp

```diff
@@ -343,17 +314,17 @@ SCENARIO("Check parameter constness with VERBATIM block",
         )";
 
         THEN("Variable used in VERBATIM shouldn't be marked as const") {
-            auto const generated = get_instance_structure(nmodl_text);
+            auto const generated = get_coreneuron_cpp_code(nmodl_text);
             std::string expected_code = R"(
                 /** all mechanism instance variables and global variables */
                 struct IntervalFire_Instance  {
                     double* invl{};
                     const double* burst_start{};
                     double* v_unused{};
                     IntervalFire_Store* global{&IntervalFire_global};
-                };
-            )";
-            REQUIRE(reindent_text(generated) == reindent_text(expected_code));
+                };)";
+            expected_code = reindent_text(expected_code, 1);
```

Comment:
It's slightly worrisome that we do this in the first place, i.e. using a string comparison to check the code...

Reply:
off-topic.